healthyregions/SDOHPlace
# #181 ,#212: Created the new UI for share link and fix the html rendering issue on description

This PR addresses issues #212 and #181. It introduces a new UI for the "Shared" function and generates a placeholder link if the description is not yet complete.

## How to Test
1. Go to the search page and select "Neighborhood Atlas."

2. You should see that the description has been replaced with "the Link." This is because the data source for this description is not yet complete in Solr, so the HTML cannot be rendered like other result items. I have added temporary code in `headerRow.tsx` to create this placeholder link, which can be removed once the data is complete:
<img width="466" alt="Screenshot 2024-08-26 at 5 15 56 PM" src="https://github.com/user-attachments/assets/ea713b2c-0dc5-4ab9-b096-275da4fccf03">

3. Click the "Share" button, and you should see the new UI style:
<img width="1232" alt="Screenshot 2024-08-26 at 5 16 17 PM" src="https://github.com/user-attachments/assets/50ba07fc-0b44-4fdc-af81-71a5976d5e68">

4. Click the "Copy" icon in the "Shared" panel. You should see a notification indicating that the link has been copied.

5. Paste the link to a new window. You should see the same page appears.
